hw/ip/otp_ctrl/data/otp_ctrl.hjson

```diff
@@ -452,165 +491,124 @@
-          writing to !!SECRET_DIGEST_CALC, the digest becomes visible in this CSR after a reset.
-          '''
-        }
-      ]
-    }
+    { multireg: {
+        name:     "CREATOR_SW_CFG_DIGEST",
+        desc:     '''
+                  Integrity digest for the creator software config partition.
+                  The integrity digest is 0 by default. Software must write this
+                  digest value via the direct access interface in order to lock the partition.
+                  After a reset, write access to the !!OWNER_SW_CFG partition is locked and
+                  the digest becomes visible in this CSR.
+                  ''',
+        count:     "2",
+        swaccess:  "ro",
+        hwaccess:  "hwo",
+        hwext:     "true",
+        cname:     "WORD",
+        fields: [
+          { bits: "31:0"
+          }
+        ]
+      }
+    },
+    { multireg: {
+        name:     "OWNER_SW_CFG_DIGEST",
+        desc:     '''
+                  Integrity digest for the owner software config partition.
+                  The integrity digest is 0 by default. Software must write this
+                  digest value via the direct access interface in order to lock the partition.
+                  After a reset, write access to the !!OWNER_SW_CFG partition is locked and
+                  the digest becomes visible in this CSR.
+                  ''',
+        count:     "2",
+        swaccess:  "ro",
+        hwaccess:  "hwo",
+        hwext:     "true",
+        cname:     "WORD",
+        fields: [
+          { bits: "31:0"
+          }
+        ]
+      }
+    },
+    { multireg: {
+        name:     "HW_CFG_DIGEST",
+        desc:     '''
+                  Hardware config partition integrity digest.
+                  The integrity digest is 0 by default. The digest calculation can be triggered via the !!DIRECT_ACCESS_CMD.
+                  After a reset, the digest then becomes visible in this CSR, and the corresponding partition becomes write-locked.
+                  ''',
+        count:     "2",
+        swaccess:  "ro",
+        hwaccess:  "hwo",
+        hwext:     "true",
+        cname:     "WORD",
+        fields: [
+          { bits: "31:0"
+          }
+        ]
+      }
+    },
+    { multireg: {
+        name:     "SECRET0_DIGEST",
+        desc:     '''
+                  Integrity digest for the SECRET0 partition.
+                  The integrity digest is 0 by default. The digest calculation can be triggered via the !!DIRECT_ACCESS_CMD.
+                  After a reset, the digest then becomes visible in this CSR, and the corresponding partition becomes write-locked.
+                  ''',
+        count:     "2",
+        swaccess:  "ro",
+        hwaccess:  "hwo",
+        hwext:     "true",
+        cname:     "WORD",
+        fields: [
+          { bits: "31:0"
+          }
+        ]
+      }
+    },
+    { multireg: {
+        name:     "SECRET1_DIGEST",
```

Comment:
is it still the case that this is calculated over the scrambled contents? Not the raw one right?

Reply:
jep that is correct.